setup.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 2.2.0
+current_version = 3.0.0
```

Comment:
>3.0.0 [](start = 18, length = 5)

Please release -rc version first, according to OneNote instructions. We will use the RC version for testing before releasing the major version. To release a new version the version reported from code needs to match with the git tag.

Reply:
I thought the -rc was only done on the git tag and not in the codebase.